aws/ecs_cluster_test.go

```diff
@@ -0,0 +1,104 @@
+package aws
+
+import (
+	"testing"
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/gruntwork-io/cloud-nuke/util"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const region = "eu-west-1"
+
+// Test we can create a cluster, tag it, and then find the tag
+func TestCanTagEcsClusters(t *testing.T) {
+	t.Parallel()
+
+	awsSession, err := session.NewSession(&awsgo.Config{
+		Region: awsgo.String(region),
+	})
+	require.NoError(t, err)
+
+	cluster := createEcsFargateCluster(t, awsSession, util.UniqueID())
+	defer deleteEcsCluster(awsSession, cluster)
+
+	tagValue := time.Now().UTC().Format(time.RFC3339)
```

Comment:
I notice you're tagging the cluster with an explicitly formatted timestamp here in the test, but not the module code above... I suspect you'll want to do it the same way in both places. You should prob create `formatTimestampTag` and `parseTimestampTag` methods that both the implementation code and test code use.

Reply:
that's also a good point, thank you 👍